src/material/schematics/ng-update/upgrade-rules/hammer-gestures-v9/hammer-gestures-rule.ts

```diff
@@ -178,7 +178,8 @@ export class HammerGesturesRule extends MigrationRule<null> {
             'The HammerJS v9 migration for Angular Components detected that HammerJS is ' +
             'manually set up in combination with references to the Angular Material gesture ' +
             'config. This target cannot be migrated completely, but all references to the ' +
-            'deprecated Angular Material gesture have been removed.');
+            'deprecated Angular Material gesture have been removed. Read more here: ' +
+            'https://git.io/ng-material-v9-hammer-ambiguous-usage');
```

Comment:
I used `git.io` (which is an official Github service) since I don't want to show +150 chars for a perma-link. Happy to switch to something else if we can generate something like `g.co/ng`, or just use a real perma-link.